extension/parquet/include/parquet_reader.hpp

```diff
@@ -96,7 +96,8 @@ class ParquetReader {
 	unique_ptr<ColumnReader> root_reader;
 
 public:
-	void InitializeScan(ParquetReaderScanState &state, vector<idx_t> groups_to_read);
+	void InitializeScan(ParquetReaderScanState &state, vector<idx_t> groups_to_read,
+	                    unordered_set<string> *used_columns);
```

Comment:
In general we use `case_insensitive_set_t` and `case_insensitive_map_t` because column names are case-insensitive in DuckDB, I'm not sure if it matters here since these are internals, but it's likely better to use `case_insensitive_set_t` here to avoid potential issues

Reply:
Not sure if this is exactly what you meant, but I changed all `unordered_set<string>` to `case_insensitive_set_t` now